src/Razor/test/Microsoft.VisualStudio.LanguageServerClient.Razor.Test/DefaultRazorLanguageServerCustomMessageTargetTest.cs

```diff
@@ -200,7 +199,8 @@ public async Task RazorRangeFormattingAsync_ValidRequest_InvokesLanguageServer()
         var outputWindowLogger = Mock.Of<IOutputWindowLogger>(MockBehavior.Strict);
 
         var documentSynchronizer = GetDocumentSynchronizer(GetCSharpSnapshot(uri, hostDocumentSyncVersion: 1));
-        var telemetryReporter = new Mock<ITelemetryReporter>(MockBehavior.Strict);
+        var telemetryReporter = new Mock<ITelemetryReporter>(MockBehavior.Default);
```

Comment:
Note that this is banned in the Razor repo. You should always use `MockBehavior.Strict` when creating Mocks. This appears in the `BannedSymbols.txt` file that gets linked into every project (though it doesn't appear to cause failures at build time).

https://github.com/dotnet/razor/blob/ec251d924953fb6900ef92ba9202e05256c2064e/eng/BannedSymbols.txt:

```
﻿M:Moq.Mock`1.#ctor; Pass MockBehavior.Strict explicitly to the constructor for Mock<T>
M:Moq.Mock.Of``1(); Pass MockBehavior.Strict explicitly to Mock.Of<T>
M:Moq.Mock.Of``1(System.Linq.Expressions.Expression{System.Func{``0,System.Boolean}}); Pass MockBehavior.Strict explicitly to Mock.Of<T>
```